src/utils/getConfigValue.ts

```diff
@@ -0,0 +1,11 @@
+import config from "../config.json";
+
+function getKeyValue(key: string, obj: Record<string, any>) {
+	return obj[key];
+}
+
+function getConfigValue(key: string) {
```

Comment:
Some TypeScript trickery. The interpreter wouldn't let me access `obj[key]` directly without the `getKeyValue()` wrapper.